bedidx.c

```diff
@@ -324,12 +333,16 @@ void *bed_read(const char *fn)
     //bed_unify(h);
     return h;
  fail:
+    save_errno = errno;
     fprintf(stderr, "[bed_read] Error reading %s : %s\n", fn, strerror(errno));
```

Comment:
All existing callers of `bed_read()` follow its failure with a call to `print_error_errno()`. So this `fprintf` is redundant — so `fail_no_msg` can be folded into `fail`. With less need for errno juggling :smile: